src/archivematicaCommon/lib/archivematicaFunctions.py

```diff
@@ -534,3 +535,32 @@ def relative_path_to_aip_mets_file(uuid, current_path):
     mets_name = "METS.{}.xml".format(uuid)
     mets_path = "{}/data/{}".format(package_name_without_extensions, mets_name)
     return mets_path
+
+
+def retrieve_storage_location_description(aip_location_slug, logger=None):
```

Comment:
I'm curious about why this is in `archivematicaFunctions.py`. I think I would have expected it to be in `storageService.py`.

Reply:
I'll have time to change this if the logic doesn't track, so just let me know. My rationale was simply that while something from the storage service is needed here the function isn't responsible for communicating with the storage service which I think the storage service functions should do. Here we've got something more specific to the Archivematica application itself. It's not a perfect abstraction mind.

Reply:
Right, I see where you're coming from. My thinking was that this seems very similar to the [filter_packages](https://github.com/artefactual/archivematica/blob/5d46475461c49997464338adc2be538f4e66a70d/src/archivematicaCommon/lib/storageService.py#L567) function from the recent reindexing work, which similarly does not directly call the Storage Service itself but handles the response of such a call. I do see that the heavy lifting (and Storage Service communication) is being done by `storage_service.location_description_from_slug`.

I don't feel strongly that this needs to change, but I am trying to determine (largely for myself) how to think about separation of functions between these modules moving forward.

Reply:
@tw4l that's a really good point, and I did make that suggestion so let me be consistent and move it :slightly_smiling_face: and perhaps it's something we can figure out more in future.

Reply:
Done! 

> but I am trying to determine (largely for myself) how to think about separation of functions between these modules moving forward.

It's a good point and honestly I don't know. I flipped-flopped before submitting the PR. And I wasn't thinking too much about that other function but it's useful to think about in hindsight. I feel the consistency is better now at least.